examples/lb/message.go

```diff
@@ -1,13 +1,18 @@
 package main
 
+import "github.com/renproject/phi"
+
 // Init is a message that signals a worker to start work.
-type Init struct{}
+type Init struct {
+	Responder chan phi.Message
+}
 
 // IsMessage implements the `phi.Message` interface.
 func (Init) IsMessage() {}
 
 // Done is a message to signal that a worker is done.
-type Done struct{}
+type Done struct {
```

Comment:
Maybe keep the old formatting to remain consistent with the other files/examples.
```suggestion
type Done struct{}
```
For example: this is how it is formatted in maxnum/message.go